circle.yml

```diff
@@ -54,8 +54,8 @@ jobs:
           name: Selenium Start
           background: true
           command: |
-            ./node_modules/.bin/webdriver-manager update
-            ./node_modules/.bin/webdriver-manager start
+            ./node_modules/.bin/webdriver-manager update --versions.chrome=2.43
```

Comment:
Why do we need to pin the chromedriver version? Is Circle stuck on Chrome 69 for some reason?

Reply:
Yes CircleCI is giving me problems. This could be resolved if we could download and run a Chromium  version.  For now this is the fix.